ts-rs/Cargo.toml

```diff
@@ -35,11 +35,13 @@ smol_str-impl = ["smol_str"]
 serde-json-impl = ["serde_json"]
 no-serde-warnings = ["ts-rs-macros/no-serde-warnings"]
 import-esm = []
+tokio-impl = ["tokio"]
 
 [dev-dependencies]
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1"
 chrono = { version = "0.4", features = ["serde"] }
+tokio = { version = "1.40", features = ["sync", "rt"] }
```

Comment:
Can we chose a somewhat older version of tokio? I think these types already existed in 1.0.